src/AppRoutes.tsx

```diff
@@ -13,6 +13,8 @@ import { useAnalytics } from 'hooks/Utils';
 import Companies from 'pages/Companies';
 import EventDetails from 'pages/EventDetails';
 import Events from 'pages/Events';
+import Gallery from 'pages/Gallery';
+import GalleryDetails from 'pages/GalleryDetails';
```

Comment:
```suggestion
const Gallery = lazy(() => import(/* webpackChunkName: "gallery" */ 'pages/Gallery'));
const GalleryDetails = lazy(() => import(/* webpackChunkName: "gallery_details" */ 'pages/GalleryDetails'));
```

Disse kan importeres lazily ettersom de ikke brukes hver gang brukeren besøker siden